action.yml

```diff
@@ -0,0 +1,34 @@
+name: 'Setup ecm-distro-tools'
+description: 'Installs ecm-distro-tools scripts to the PATH.'
+inputs:
+  version:
+    description: 'The tag of the ecm-distro-tools release to install.'
+    required: true
+runs:
+  using: 'composite'
+  steps:
+  - shell: bash
+    run: |
+      TAG="${{ inputs.version }}"
```

Comment:
Can you run shellcheck on this to make sure it adheres to that and our script standards?

Reply:
Ran it through shellcheck and committed a change.